src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -381,26 +381,20 @@ public static void writeBundle(
                     CountingOutputStream countingOs = new CountingOutputStream(outputStream);
                     ZipArchiveOutputStream binaryOut =
                             new ZipArchiveOutputStream(new BufferedOutputStream(countingOs, 16384))) {
-                // Get the filter to be used
-                Optional<ContentFilter> maybeFilter = getContentFilter(false);
-                // Recalculate the mappings and stop words and save it to disk
-                maybeFilter.ifPresent(SupportPlugin::reloadAndSaveMappings);
+                ContentFilter filter = getDefaultContentFilter(true);
 
-                // Generate the content of the manifest.md going trough all the components which will be included. It
-                // also returns the contents to include. We pass maybeFilter to filter the names written in the manifest
+                // Generate the content of the manifest.md going through all the components which will be included. It
+                // also returns the contents to include. We pass a filter to filter the names written in the manifest
                 appendManifestHeader(manifest);
                 long startTime = System.currentTimeMillis();
                 List<Content> contents =
-                        appendManifestContents(manifest, errorWriter, components, componentConsumer, maybeFilter);
+                        appendManifestContents(manifest, errorWriter, components, componentConsumer, filter);
                 LOGGER.log(
                         Level.FINE,
                         "Took " + (System.currentTimeMillis() - startTime) + "ms to process all components");
                 contents.add(new UnfilteredStringContent("manifest.md", manifest.toString()));
 
-                Optional<FilteredOutputStream> maybeFilteredOut =
-                        maybeFilter.map(filter -> new FilteredOutputStream(binaryOut, filter));
-                OutputStream textOut =
-                        maybeFilteredOut.map(OutputStream.class::cast).orElse(binaryOut);
```

Comment:
@Dohbedoh For what it's worth, while debugging issues with a log file using a support bundle, after a bit of confusion I think that this PR had the side effect of converting Windows line endings in files in the support bundle to Unix line endings, even when anonymization is disabled, at least in some cases.

I think the key difference is that in the old code, if anonymization was disabled, we just passed everything through via binary streams with no processing. With the new code, even if anonymization is disabled, we pass everything through `FilteredOutputStream`, which decodes the binary data into text, applies the filter, and then re-encodes the text, and I guess somewhere along the way line endings get normalized. Beyond the line endings issue though, I suspect performance might also be worse due to the increased processing overhead when anonymization is disabled.

Reply:
The specific issue was actually with `BaseFileContent`, see https://github.com/jenkinsci/support-core-plugin/pull/502/files#r1492687323 and https://github.com/jenkinsci/support-core-plugin/pull/509.